pkg/runtime/wfengine/component.go

```diff
@@ -156,32 +162,32 @@ func (c *workflowEngineComponent) Get(ctx context.Context, req *workflows.Workfl
 		return nil, fmt.Errorf("failed to get workflow metadata for '%s': %w", req.InstanceID, err)
 	} else {
 		res := &workflows.StateResponse{
-			WFInfo: workflows.WorkflowReference{
-				InstanceID: req.InstanceID,
-			},
-			StartTime: metadata.CreatedAt.Format(time.RFC3339),
-			Metadata: map[string]string{
-				"dapr.workflow.name":           metadata.Name,
-				"dapr.workflow.runtime_status": getStatusString(int32(metadata.RuntimeStatus)),
-				"dapr.workflow.input":          metadata.SerializedInput,
-				"dapr.workflow.custom_status":  metadata.SerializedCustomStatus,
-				"dapr.workflow.last_updated":   metadata.LastUpdatedAt.Format(time.RFC3339),
+			Workflow: &workflows.WorkflowState{
+				InstanceID:    req.InstanceID,
+				WorkflowName:  metadata.Name,
+				CreatedAt:     metadata.CreatedAt,
+				LastUpdatedAt: metadata.LastUpdatedAt,
+				RuntimeStatus: getStatusString(int32(metadata.RuntimeStatus)),
+				Properties: map[string]string{
+					"dapr.workflow.input":         metadata.SerializedInput,
```

Comment:
Can these be constants?

Reply:
They can, but is that what we want when they're only used in one place? I didn't see an advantage to using constants for these, only disadvantages.

Reply:
The advantage would be if they were used in unit tests too. Otherwise, ok

Reply:
My counter to that is duplicating these values in unit tests helps prevent accidental breaking changes (a unit test is a bit more representative of customer code this way). :)